rust/blockstore/src/arrow/provider.rs

```diff
@@ -377,11 +377,6 @@ impl SparseIndexManager {
         }
     }
 
-    pub fn create(&self, id: &Uuid) -> SparseIndex {
```

Comment:
not being used anywhere?

Reply:
will clean up separately, as it was not added in this PR and I don't want to retrigger tests